portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,34 +26,47 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    private Map<String, String> comments = new HashMap<String, String>();
-
-    @Override
-    public void init() {
-        comments.put("Mario", "Hi!");
-        comments.put("Mario's imaginary friend", "Hi! What are you going?");
-        comments.put("Mario", "Just learning how to use JSON");
-        comments.entrySet().forEach(entry->{
-            System.out.println(entry.getKey() + ": " + entry.getValue());  
-        });
-    } 
-
-    @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String json = convertToJson(comments);
-        System.out.println("Sending string: " + json);
-        response.setContentType("application/json;");
-        response.getWriter().println(json);
-    }
+  // IMPROVE: redundently storing display names
```

Comment:
remove empty space

Reply:
The empty space after the '//'?

Reply:
I thought there was an empty line. Maybe just the diff side by side is messing me up